src/components/policy/policy_external/src/update_status_manager.cc

```diff
@@ -62,6 +62,8 @@ UpdateStatusManager::~UpdateStatusManager() {
 void UpdateStatusManager::ProcessEvent(UpdateEvent event) {
   sync_primitives::AutoLock lock(status_lock_);
   current_status_->ProcessEvent(this, event);
+  last_processed_event_ = event;
+  LOG4CXX_DEBUG(logger_, "last_processed_event_ = " << last_processed_event_);
```

Comment:
This duplicate log (appears in `DoTransition()` as well) might cause problems with debugging, should probably be removed